File: factorial.py
def factorial(n):

    fact = 1
    for i in range(1, n + 1):
        fact *= i
    
    return fact

def main():

    inputs = [0, 1, 5]

    for input in inputs:
        print("Input: ", input)
        print("Output :", factorial(input))

main()

File: Factorial.java
public class Factorial{

    int run(int n){

        int x = 1;

        if (n == 0){
            return 1;
        }
        else{
            for (int i = 1; i <= n; i++){
                x *= i;
            }

            return x;
        }
    }

    public static void main(String[] args){
        Factorial example = new Factorial();
        System.out.println(example.run(6));
    }
}